sdk/typescript/faucet/package.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
You'll need to add this to the files section of the root package.json or it won't make it into the package when it's published